<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BerichteOrk</class>
 <widget class="QMainWindow" name="BerichteOrk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wochenberichte</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/accessories-text-editor.png</normaloff>:/images/images/accessories-text-editor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gewählte Woche:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="selectedWeekLabel">
        <property name="text">
         <string>Keine Woche gewählt...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="wochenTree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="jumpToDateButton">
          <property name="text">
           <string>Zu einem Datum springen...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Betrieb</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableView" name="betriebView">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>358</width>
                 <height>17</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addBetriebButton">
               <property name="text">
                <string>Neuen Eintrag erstellen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeBetriebButton">
               <property name="text">
                <string>Markierte Einträge entfernen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Schule</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableView" name="schuleView"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>418</width>
                 <height>17</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addSchuleButton">
               <property name="text">
                <string>Neuen Eintrag erstellen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeSchuleButton">
               <property name="text">
                <string>Markierte Einträge entfernen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionGeneratePDF"/>
   <addaction name="separator"/>
   <addaction name="actionBeenden"/>
  </widget>
  <action name="actionGeneratePDF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/application-pdf.png</normaloff>:/images/images/application-pdf.png</iconset>
   </property>
   <property name="text">
    <string>&amp;PDF Datei erzeugen...</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/system-log-out.png</normaloff>:/images/images/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="iconText">
    <string>&amp;Beenden</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/document-open.png</normaloff>:/images/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Öffnen...</string>
   </property>
   <property name="iconText">
    <string>&amp;Öffnen</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/document-new.png</normaloff>:/images/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Neues Berichtsheft</string>
   </property>
   <property name="iconText">
    <string>&amp;Neues Berichtsheft</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>BerichteOrk</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
